test.js

```diff
@@ -37,8 +37,7 @@ const tests = {
 }
 
 const format = {compact: true, semicolons: false}
-const generate = ast => escodegen.generate(ast, {format})
```

Comment:
Not sure why, but currently the tests pass because this const is available in lave.js:65.